Update Kaptain configuration page (#4338)
mesosphere/dcos-docs-site: pages/dkp/kaptain/2.0.0/configuration/index.md

<!-- markdownlint-disable -->
[//]: # "WARNING: This page is auto-generated and should not be modified directly."

This page lists Kaptain configuration properties that can be adjusted during the initial installation or by updating the existing instance of Kaptain. These parameters are typically set in `values.yaml`.

### Kaptain Chart Values
| Key | Type | Default | Description |
|-----|------|---------|-------------|
| global.clusterDomainName | string | `"cluster.local"` | Kubernetes cluster domain name |
| global.userIdHeader | string | `"kubeflow-userid"` | Name of the HTTP header containing the information about authenticated user |
| global.userIdPrefix | string | `""` | Prefix for the 'userIdHeader' header value |
| global.workspace | string | `"kommander"` |  |
| ingress.enabled | bool | `true` | Enable ingress |

> | ingress.externalDexClientId | string | `""` | Dex client id to use when authenticating with external cluster |

Reviewer: ```suggestion
| ingress.externalDexClientId | string | `""` | Dex client ID to use when authenticating with external cluster |
```

| core.notebook.images[2] | string | `"mesosphere/kubeflow-dev:22caa41c-jupyter-spark-3.0.0-horovod-0.24.2-pytorch-1.11.0"` | JupyterLab with Pytorch, Spark and Horovod pre-installed |
| core.notebook.images[3] | string | `"mesosphere/kubeflow-dev:d431dd5f-jupyter-spark-3.0.0-horovod-0.24.2-pytorch-1.11.0-gpu"` | JupyterLab with Pytorch, CUDA, Spark and Horovod pre-installed with GPU support |
| core.notebook.images[4] | string | `"mesosphere/kubeflow-dev:44625480-jupyter-spark-3.0.0-horovod-0.24.2-mxnet-1.9.0"` | JupyterLab with MXNet, Spark and Horovod pre-installed |
| core.notebook.images[5] | string | `"mesosphere/kubeflow-dev:e6c6aec9-jupyter-spark-3.0.0-horovod-0.24.2-mxnet-1.9.0-gpu"` | JupyterLab with Tensorflow, CUDA, Spark and Horovod pre-installed with GPU support |
| core.notebook.tolerationGroups | list | `[]` | Pod toleration group configurations for Notebook servers |
| core.notebook.affinityConfig | list | `[]` | Pod affinity configurations for Notebook servers |
| core.notebook.enableCulling | bool | `false` | Enables scale down idling notebooks to freeing up the allocated resources. |

> | core.notebook.idleTimeMinutes | int | `1440` | The period of time a notebook can stay idle before it gets culled (scaled down) |

Reviewer: ```suggestion
| core.notebook.idleTimeMinutes | int | `1440` | Period of time a notebook can stay idle before it gets culled (scaled down) |
```

| core.notebook.cullingCheckPeriodMinutes | int | `1` | Notebook status check period |

> | core.disableAntiAffinity | bool | `false` | Enables single-node installation. NOT FOR PRODUCTION USE! Disabling anti-affinity allows installing Kaptain on a smaller number of nodes (<3) or on a single node. This installation mode must be used for evaluation purposes only. Not recommended for production use! No data integrity guarantees in case of a hardware failure! |

Reviewer: ```suggestion
| core.disableAntiAffinity | bool | `false` | Enables single-node installation. **DO NOT USE IN PRODUCTION ENVIRONMENTS!** Disabling anti-affinity allows installing Kaptain on a smaller number of nodes (less than 3) or on a single node. This installation mode must be used **for evaluation purposes only**. No data integrity guarantees in case of a hardware failure! |
```

| core.db.nodes.memory | string | `"2G"` | Memory per single database node in the MySQL cluster |
| core.db.nodes.cpu | int | `2` | CPU per single database node in the MySQL cluster |
| core.db.proxy.count | int | `2` | Number of proxy instances for the HA MySQL cluster |
| core.db.proxy.memory | string | `"1G"` | Memory per single proxy instance in the MySQL cluster |
| core.db.proxy.cpu | int | `1` | CPU per single proxy instance in the MySQL cluster |
| core.db.backup.enabled | bool | `false` | Enable backup for the MySQL cluster |
| core.db.backup.bucket | string | `""` | Backup bucket name for the MySQL cluster |
| core.db.backup.region | string | `""` | Backup bucket region name for the MySQL cluster |
| core.db.backup.secretName | string | `""` | Name of the secret with access credentials for the MySQL cluster backup bucket |
| core.db.backup.endpointUrl | string | `""` | Custom endpoint URL for the MySQL cluster backup bucket |
| core.db.backup.restoreSource | string | `""` | Full path to the backup folder, for example `s3://mysql-store-backup/kaptain-mysql-store-2021-04-05-21:47:24-full`. |

> | core.minio.serverPools | int | `1` | Number of MinIO server pools. |

Reviewer: ```suggestion
| core.minio.serverPools | int | `1` | Number of MinIO server pools |
```

| core.minio.volumesPerServer | int | `2` | The number of Persistent Volume Claims to generate for each MinIO server pod in the pool. |
| core.minio.storageCapacity | string | `"40Gi"` | The total capacity of a single MinIO server pool. |
| core.minio.volumeStorageClass | string | `""` | StorageClass name to be used for PVCs in MinIO server pool. |
| core.minio.installMinioOperator | bool | `false` | Install Minio Operator to target namespace. Must be disabled in case there is another instance of the operator running in the cluster. |
| core.pipelines.bucketName | string | `"mlpipeline"` |  |
| core.pipelines.cacheDb | string | `"cachedb"` |  |
| core.pipelines.dbConMaxLifeTimeSec | string | `"300s"` |  |
| core.pipelines.mlmdDB | string | `"metadb"` |  |
| core.pipelines.pipelineDB | string | `"mlpipeline"` |  |
| core.pipelines.defaultPipelineRunnerServiceAccount | string | `"default-editor"` |  |
| core.pipelines.objectStoreHost | string | `"minio"` |  |
| core.pipelines.objectStorePort | int | `80` |  |
| core.pipelines.workflowsTTLSecondsAfterFinish | int | `86400` | The TTL for Argo Workflow (a single run of a pipeline) to persist after completion (default: 24 hours). |

> | core.pipelines.enableMultiUserSupport | bool | `true` | Enable multi-user support for Pipelines |

Reviewer: ```suggestion
| core.pipelines.enableMultiUserSupport | bool | `true` | Enables multi-user support for Pipelines |
```

> | kserve.enabled | bool | `true` | Enable KServe |

Reviewer: ```suggestion
| kserve.enabled | bool | `true` | Enables KServe |
```
